src/components/notice/notice-write/notice-write.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+import { Button } from '@/components/ui/button';
+import { useSearchParams } from 'react-router-dom';
+// import NoticeEditor from './notice-editor';
+import { useState, useEffect } from 'react';
+import { NotepadText } from 'lucide-react';
+import NoticeNav from '../notice-nav.tsx/notice-nav';
+import NoticeEditor from './notice-editor';
+import { CATEGORY_LIST } from '@/constants/notice/notice-category';
+
+const categoryList = CATEGORY_LIST;
+
+export default function NoticeWrite() {
+  const [title, setTitle] = useState('');
+  const [searchParams, setSearchParams] = useSearchParams();
+  const selectedCategory = searchParams.get('category') ?? '일반';
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+
+  const handleCategoryChange = (category: string) => {
+    setSearchParams((prevSearchParams) => {
+      if (category === 'all') {
+        prevSearchParams.delete('category');
+      } else {
+        prevSearchParams.set('category', category);
+      }
+      return prevSearchParams;
+    });
+  };
+
+  useEffect(() => {
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth',
+    });
+  }, []);
+  return (
+    <div className='h-full w-full max-w-[1260px] pt-[100px] px-5 xl:px-0 mx-auto text-center'>
+      <div className='w-full  text-center'>
+        <div className='flex justify-start  bitems-center  gap-1 w-full pb-[18px]'>
```

Comment:
```suggestion
        <div className='flex justify-start items-center  gap-1 w-full pb-[18px]'>
```

`bitems-center`는 오타죠?